extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -126,14 +136,19 @@ export class AmpSidebar extends AMP.BaseElement {
    */
   open_() {
     this.viewport_.disableTouchZoom();
-    this.mutateElement(() => {
-      this.viewport_.addToFixedLayer(this.element);
+    this.vsync_.mutate(() => {
       setStyles(this.element, {
         'display': 'block',
       });
+      this.viewport_.addToFixedLayer(this.element);
       this.openMask_();
-      this.element.setAttribute('open', '');
-      this.element.setAttribute('aria-hidden', 'false');
+      this.vsync_.mutate(() => {
+        this.element.setAttribute('open', '');
```

Comment:
Add a comment, e.g. "Start animation in a separate vsync due to display: none." or something like this.